package/MDAnalysis/analysis/psa.py

```diff
@@ -61,8 +61,7 @@
     Path,
     PSAPair,
     PSAnalysis,
-)
-
+) 
```

Comment:
here and everywhere else this is done, please don't remove this extra blank line - it's there for pep8 reasons